app/config/bootstrap/cache.php

```diff
@@ -83,15 +83,15 @@
 		if (!(($connection = Connections::get($name)) instanceof Database)) {
 			continue;
 		}
-		Filters::apply($connection, 'describe', function($params, $chain) use ($name) {
```

Comment:
Actually changing $chain to $next is the only thing needed after looking through this.